src/lib/tests/site-language.test.ts

```diff
@@ -5,6 +5,9 @@ import { app } from 'electron';
 import { getPreferredSiteLanguage } from 'src/lib/site-language';
 
 jest.unmock( 'fs-extra' );
+jest.mock( 'src/storage/user-data', () => ( {
+	loadUserData: jest.fn().mockResolvedValue( { locale: undefined } ),
+} ) );
```

Comment:
Without this mock, this test outputs a bunch of warnings like this:

```
  console.error
    Failed to load file /path/to/app/appData/App Name/appdata-v1.json: Error: ENOENT: no such file or directory, open '/path/to/app/appData/App Name/appdata-v1.json'

      79 | 			};
      80 | 		}
    > 81 | 		console.error( `Failed to load file ${ sanitizeUserpath( filePath ) }: ${ err }` );
         | 		        ^
      82 | 		throw err;
      83 | 	}
      84 | }

      at src/storage/user-data.ts:81:11
          at Generator.throw (<anonymous>)
      at rejected (src/storage/user-data.ts:29:65)
```

Reply:
Also, maybe let's add it as a comment to code?

Reply:
Fair 👍